views/index.ejs

```diff
@@ -12,11 +11,11 @@
   <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
   <script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.10/angular-ui-router.js"></script>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+
 
   <!-- libraries -->
   <script src="/javascripts/libraries/ui-bootstrap-tpls-0.13.0.min.js"></script>
-
+<script src="https://apis.google.com/js/client:platform.js" async defer> </script>
```

Comment:
This is already present na ?  
Why are you requesting for the same file again ? This is being loaded.